### PR TITLE
Added task for deploying Barbican

### PR DESCRIPTION
**Pull request by jsuchome**
~Not yet tested~

Tested with few CLI options (https://docs.openstack.org/python-barbicanclient/latest/cli/details.html)
, seems to work without issues

### CODE REVIEW

File: playbooks/roles/deploy-osh/tasks/main.yml [hunk 1]
```diff
@@ -417,3 +417,19 @@
   tags:
     - heat
     - run
+
+- name: Deploy Barbican
+  import_tasks: component-install.yml
+  vars:
+    _component_runcommand: "./tools/deployment/multinode/160-barbican.sh"
+    _component_run_env:
+      OSH_EXTRA_HELM_ARGS_HEAT: >
```
Comment (Itxaka) [inline playbooks/roles/deploy-osh/tasks/main.yml]: same as my tempest change, copy&paste gone wrong :P

Comment (jsuchome) [reply playbooks/roles/deploy-osh/tasks/main.yml]: I actually fixed it locally, only forgot to push :-)

**Comment by jsuchome** [comment]
We have to wait until image is published in dockerhub

**Comment by jsuchome** [comment]
> We have to wait until image is published in dockerhub

-> DONE, PR is ready for reviews

**Comment by toabctl** [comment]
Not sure if related, but the CI failed with `/bin/sh: /usr/bin/python: No such file or directory
`
See https://ci.suse.de/blue/organizations/jenkins/cloud-socok8s%2Fsocok8s/detail/PR-274/1/pipeline#step-25-log-1913

**Comment by rhafer** [comment]
> Not sure if related, but the CI failed with `/bin/sh: /usr/bin/python: No such file or directory`

I think this was related to the change of the Leap image for the deployer (which was reverted meanwhile): https://github.com/SUSE-Cloud/socok8s/pull/309

**Pull request merged**